pymdownx/blocks/block.py

```diff
@@ -331,10 +345,11 @@ def on_validate(self, parent):
         return True
 
     @abstractmethod
-    def on_create(self, parent):
+    def on_create(self, parent: etree.Element) -> etree.Element:
         """Create the needed element and return it."""
+        ...
```

Comment:
Remove `...`, it isn't needed when there is a docstring and is just causing test coverage issues.